core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -139,12 +139,17 @@ public static void main(String[] args) throws Exception {
         }
 
         final WorkerPool workerPool = new WorkerPool(configuration.getWorkerPoolConfiguration());
-        final MessageBus messageBus = new MessageBusImpl();
+        final MessageBus messageBus = new MessageBusImpl(configuration);
         final FunctionFactoryCache functionFactoryCache = new FunctionFactoryCache(configuration.getCairoConfiguration(), ServiceLoader.load(FunctionFactory.class));
 
         LogFactory.configureFromSystemProperties(workerPool);
         final CairoEngine cairoEngine = new CairoEngine(configuration.getCairoConfiguration(), messageBus);
         workerPool.assign(cairoEngine.getWriterMaintenanceJob());
+        final TelemetryJob telemetryJob = new TelemetryJob(configuration, cairoEngine, messageBus);
+
+        if (configuration.getTelemetryConfiguration().getEnabled()) {
```

Comment:
when telemetry is disabled it is better not to create job at all. This would meet potential user expectations such as:

- i disabled telemetry
- telemetry tables are not created

Reply:
TelemetryJob is needed even when telemetry is turned off because it is responsible for updating the `telemetry_config` table, if I remove it, telemetry will always be on from the UI perspective (because the table won't be updated). I will keep this as is.